engine/class_modules/sc_death_knight.cpp

```diff
@@ -4672,6 +4676,18 @@ struct empower_rune_weapon_t : public death_knight_spell_t
     cooldown -> duration += p -> spec.empower_rune_weapon_2->effectN( 1 ).time_value();
   }
 
+  double recharge_multiplier( const cooldown_t& cd ) const override
+  {
+    double m = death_knight_spell_t::recharge_multiplier( cd );
+
+    if ( p() -> conduits.accelerated_cold->ok() )
+    {
+      m *= 1.0 + p()->conduits.accelerated_cold->effectN( 2 ).percent();
```

Comment:
This should be a division: a recharge multiplier below 1 means it's faster, and longer when above 1.

Reply:
This one is the one mentioned above where I referenced sc_action.cpp:4917 for the implementation. Where it is of type "Modify Recharge Time% (Category) (454)", which in the apply_affecting_aura it is implemented as a multiplication.

effectN( 2 ) for accelerated_cold is a negative number ( -10 )
empower_rune_weapon rank 1 has a base cooldown of 120 seconds
empower_rune_weapon rank 2 reduces the cooldown by 15 seconds, which we can find at sc_deathknight.cpp:4676
So we have a 120 second cooldown, reduced by 15, then reduced by 10%
to find the multiplier, from spell data .percent() call we get:
accelerated_multi = 1.0 + (-0.10) = 0.90

Multiplication
So full math looks something like:

accelerated_multi = 1.0 + (-0.10) = 0.90
120 - 15 = 105
105 * accelerated_multi = 94.5

When I run with a very simple profile with multiplication version, I get the following results in the log:

1.326 Player 'PR_Death_Knight_Frost_2H' schedules execute for Action empower_rune_weapon
95.826 Player 'PR_Death_Knight_Frost_2H' schedules execute for Action empower_rune_weapon (94.5 sec diff)
190.326 Player 'PR_Death_Knight_Frost_2H' schedules execute for Action empower_rune_weapon (94.5 sec diff)

Division
For the division version, following the same math:

accelerated_multi = 1.0 + (-0.10) = 0.90
120 - 15 = 105
105 / accelerated_multi = 116.66

When I run the same simple profile, with the division version I get the following results in the log:

1.329 Player 'PR_Death_Knight_Frost_2H' schedules execute for Action empower_rune_weapon
117.995 Player 'PR_Death_Knight_Frost_2H' schedules execute for Action empower_rune_weapon (116.66 sec diff)
234.661 Player 'PR_Death_Knight_Frost_2H' schedules execute for Action empower_rune_weapon (116.66 sec diff)

Results
Based on testing, I have to conclude that this requires a multiplication, as it causes ERW to trigger every 94.5 seconds, whereas the division version is causing it to trigger every 116.66 seconds. Either that, or I am completely mis-interpreting the timestamps in the logs. Which is entirely possible.

Reply:
Yeah I was going by the description links rather than the spelldata labels and thought "+5" was the modifier for spell recharge time.
> Description      : Empower Rune Weapon recharges $s1% faster and the Haste amount is increased by |cFFFFFFFF${$s2}.1%|r.
With -10 the multiplication makes complete sense 👍

Reply:
I'll ask around if anyone has been able to test it on beta, definitely weird that the effect labels are contradicting the description.

Reply:
Yeah, tooltip is bugged :D  They swapped the values in it.  Makes it confusing.

Reply:
When I resolved this earlier I hadn't noticed your comment about getting it tested.  I agree it should get some more testing.  Looking at the spell data for this one the conduit rank data does match effect1, starting at 5.

I do expect either the description to change, or the conduit data to change to sync up.  As currently implemented,  based on the effects, it does give a similar % dps increase to the other conduits.

Conduit Id       : 79 (values=5.0, 5.5, 6.0, 6.75, 7.5, 8.25, 9.0, 9.75, 10.5, 11.25, 12.0, 12.75, 13.5, 14.25, 15.0)

Effects          :
#1 (id=836041)   : Apply Aura (6) | Add Flat Modifier (107): Spell Effect 3 (23)
                   Base Value: 5 | Scaled Value: 5 | PvP Coefficient: 1.00000 | Misc Value: 23 | Target: Self (1)
                   Affected Spells: Empower Rune Weapon (47568), Hungering Rune Weapon (207127)
                   Family Flags: 14
#2 (id=836047)   : Apply Aura (6) | Modify Recharge Time% (Category) (454)
                   Base Value: -10 | Scaled Value: -10 | PvP Coefficient: 1.00000 | Misc Value: 1614 | Target: Self (1)
                   Affected Spells (Category): Empower Rune Weapon (47568), Hungering Rune Weapon (207127)
Description      : Empower Rune Weapon recharges $s1% faster and the Haste amount is increased by |cFFFFFFFF${$s2}.1%|r.

Reply:
Had Druchaon on archerus test it.  CD/haste seen on live matches spelldata.  Tooltip is incorrect.

Reply:
Looks like latest beta update fixed the tool tip.